src/app/+dashboard/components/badges/badges.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <div class="header">
+    <p>BADGES</p>
+  </div>
+
+  <div class="badges">
+    <p id="badg-numb">4</p>
```

Comment:
It would be nice if you provide full css class/id names without different abbreviations. For example here you can rename badg-numb into badges-number. It would be more informative. Please double check all your classes and provide full class/id names instead of their shorter version.

Reply:
fixed